src/careamics/lightning/dataset_ng/data_module.py

```diff
@@ -71,8 +71,6 @@ def __init__(
         val_data_target: Optional[Any] = None,
         pred_data: Optional[Any] = None,
         pred_data_target: Optional[Any] = None,
-        image_stack_loader: ImageStackLoader,
```

Comment:
Is this overload still necessary? I think it is now the same than the first one.

In my docstring PR, I removed it so we can also ignore it here...